WordPress/Classes/Plugins/Views/PluginListItemView.swift

```diff
@@ -3,68 +3,105 @@ import SwiftUI
-            PluginIconView(slug: slug, service: service)
+            PluginIconView(slug: plugin.possibleWpOrgDirectorySlug, service: viewModel.service)
 
             VStack(alignment: .leading, spacing: 0) {
-                Text(name)
+                Text(plugin.name)
                     .lineLimit(1)
                     .font(.headline)
                     .foregroundStyle(.primary)
 
-                if !author.isEmpty {
-                    Text(Strings.author(author))
-                        .lineLimit(1)
-                        .font(.caption)
-                        .foregroundStyle(.secondary)
-                }
-
                 Group {
-                    if shortDescription.isEmpty {
+                    if plugin.shortDescription.isEmpty {
                         Text(Strings.noDescriptionAvailable)
                             .font(.system(size: descriptionFontSize).italic())
                     } else if let html = renderedDescription() {
                         Text(html)
                     } else {
-                        Text(shortDescription)
+                        Text(plugin.shortDescription)
                             .font(.system(size: descriptionFontSize))
                     }
                 }
+                .lineLimit(2)
                 .padding(.vertical, 4)
 
-                Text(Strings.version(version))
+                Text(Strings.version(plugin.version))
                     .font(.caption)
                     .foregroundStyle(.secondary)
             }
+
+            Spacer()
+        }
+        .overlay(alignment: .bottomTrailing) {
+            Menu {
+                Section {
+                    if plugin.isActive {
+                        Button(Strings.deactivate, systemImage: "bolt.slash") {
+                            Task {
+                                await viewModel.toggle(slug: plugin.slug)
+                            }
+                        }
+                    } else {
+                        Button(Strings.activate, systemImage: "bolt") {
+                            Task {
+                                await viewModel.toggle(slug: plugin.slug)
+                            }
+                        }
+                        Button(Strings.delete, systemImage: "trash") {
+                            Task {
+                                await viewModel.uninstall(slug: plugin.slug)
+                            }
+                        }
+                    }
+                }
+                .disabled(isUpdating)
+
+                if wpOrgURL != nil {
+                    Section {
+                        Button {
+                            showingSafariView = true
+                        } label: {
+                            Label(Strings.viewOnWordPressOrg, systemImage: "safari")
+                        }
+                    }
+                }
+            } label: {
+                Image(systemName: "ellipsis")
```

Comment:
I suggest moving it to the top-right corner. It looks a bit off where it is now.

Reply:
The menu should also be available using the context menu (long-press).

Reply:
I used the Apple Podcasts app's episodes list as a reference (i.e., Library -> Latest Episodes). Can you review it again and see if the ellipsis button needs to be moved?

Reply:
In works in Podcasts because it shows images in the top-right corner when it can. When there is no images, there is more vertical spacing between the details and the ellipsis. It should have more spacing or it looks off.